kubetest2/kubetest2-gce/deployer/deployer.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package deployer implements the kubetest2 GKE deployer
+package deployer
+
+import (
+	"fmt"
+
+	"k8s.io/test-infra/kubetest2/pkg/types"
+
+	"github.com/octago/sflags/gen/gpflag"
+	"github.com/spf13/pflag"
+)
+
+// Name is the name of the deployer
+const Name = "gce"
+
+type deployer struct {
+	// generic parts
+	commonOptions types.Options
+
+	RepoRoot string `desc:"The path to the root of the local kubernetes/cloud-provider/gcp repo. Necessary to call certain scripts. Defaults to the current directory. If operating in legacy mode, this should be set to the local kubernetes/kubernetes repo."`
```

Comment:
maybe in a later follow up consider including this under something like buildOptions.

Reply:
Definitely.